Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -246,7 +246,8 @@ open class MessageContentCell: MessageCollectionViewCell {
             fatalError(MessageKitError.avatarPositionUnresolved)
         }
 
-        messageContainerView.frame = CGRect(origin: origin, size: attributes.messageContainerSize)
+        let size = CGSize(width: attributes.messageContainerSize.width - avatarPadding.horizontal, height: attributes.messageContainerSize.height)
```

Comment:
The frame can't be modified here, it needs to be correct inside of the `MessagesCollectionViewLayoutAttributes`

This is where you need to be making the change:
https://github.com/MessageKit/MessageKit/blob/master/Sources/Layout/MessageSizeCalculator.swift#L227-L233

Reply:
It can't size the text properly if it doesn't know how much padding is consuming space

Reply:
Ah thats where it should be modified.